src/Administration/Resources/app/administration/src/module/sw-import-export/service/importExport.service.ts

```diff
@@ -150,7 +150,7 @@ export default class ImportExportService extends ApiService {
         );
 
         const createdLog = await this.httpClient.post('/_action/import-export/prepare', formData, {
-            headers: this.getBasicHeaders(),
+            headers: this.getBasicHeaders({'Content-Type': 'multipart/form-data'}),
```

Comment:
Fyi @seggewiss @jleifeld we discovered this crazy bug:
your change from `axios: 0.27.2` to `axios: 0.29.0` broke the import / export file upload. Reason was a slight change in the library behaviour: if the header said `application/json` it converts now to it, even if the data is `FormData`. We forgot to set the `content-type` header correctly and it used the default `application/json`, previously this worked somehow and `axios` has overriden it correctly to `multipart/form-data` before...